cocos/core/platform/debug.ts

```diff
@@ -216,7 +216,7 @@ export function _resetDebugSetting (mode: DebugMode) {
     if (mode !== DebugMode.ERROR) {
         if (EDITOR) {
             ccWarn = console.warn.bind(console);
-        } else if (console.warn.bind) {
+        } else if (console.warn.bind && (!window.oh)) { //TODO : resolve openharmony warn exception.
             // use bind to avoid pollute call stacks
```

Comment:
maybe we need to use the hilog SDK in the future
instead of console.log

Reply:
yes